javascript-data-model/order-history.js

```diff
@@ -0,0 +1,32 @@
+var orderHistory = [
+  {
+    item: 'JavaScript for impatient programmers',
+    author: 'Rauschmayer, Dr. Axel',
+    price: '$31.55',
+    orderPlaced: 'August 4, 2020',
+    orderDelivered: 'Aug 8, 2020'
+  },
+  {
+    item: 'The Timeless Way of Building',
+    author: 'Alexander, Christopher',
+    price: '$41.33',
+    orderPlaced: 'July 19, 2020',
+    orderDelivered: 'Jul 20, 2020'
+  },
+  {
+    item: 'Gamecube Controller Adapter Super Smash Bros Switch Gamecube Adapter for WI U, PC Support Turbo and Vibration Features No Driver and No Lag Gamecube Adapter',
+    price: '$15.98',
+    orderPlaced: 'July 4, 2020',
+    orderDelivered: 'Jul 7, 2020'
+  },
+  {
+    item: ['GameCube Controller Super Smash Bros.Edition(Nintendo Switch)', 'The Art of Sql'],
+    author: 'Faroult, Stephane',
+    price: ['$94.95', '$33.99'],
```

Comment:
This format is different than in your other order objects, remember that the format needs to be the same across all order objects. If one object stores its data differently than another, it will lead to **_many_** bugs 🤢